keccak256/src/gates/base_conversion.rs

```diff
@@ -148,7 +148,7 @@ impl<F: FieldExt> BaseConversionConfig<F> {
                         region.constrain_equal(input_acc_cell, input_coef_cell)?;
                         region.constrain_equal(output_acc_cell, output_coef_cell)?;
                     } else if offset == input_coefs.len() - 1 {
-                        region.constrain_equal(input_acc_cell, input.0)?;
+                        //region.constrain_equal(input_acc_cell, input.0)?;
```

Comment:
Is this meant to be commented out?

Reply:
It is. We reviewed that with @ChihChengLiang and this leads to a Permutation err as we copy from different cells different rounds.

We should maybe look into this a bit more.

Reply:
per offline discussion with @CPerezz, the copy constraint is required, but it might be difficult to enable in this PR. We can enable it in a future PR.